docs/proposals/operator-hub.md

```diff
@@ -0,0 +1,197 @@
+the services (basically Custom Resources in Operator framework terminology) on
+an OpenShift or Kubernetes cluster. Considering the ongoing `kclient` and DevFile
+efforts, we intend to add support for Kubernetes from day 1. However, there
+will be certain prerequisites that need to be satisfied in case of Kubernetes.
+
+Table of Contents
+- [Which Operators do we support](#which-operators-do-we-support)
+- [Installing Operators](#installing-operators)
+- [List the services/operators](#list-the-servicesoperators)
+  - [Both Service Catalog and Operator Hub are enabled](#both-service-catalog-and-operator-hub-are-enabled)
+  - [Only Service Catalog is enabled](#only-service-catalog-is-enabled)
+  - [Only Operator Hub is enabled](#only-operator-hub-is-enabled)
+  - [Kubernetes](#kubernetes)
+- [Link components with operands](#link-components-with-operands)
+- [Listing the services](#listing-the-services)
+- [Describe deployed services](#describe-deployed-services)
+- [Delete the service](#delete-the-service)
+- [Testing on Kubernetes](#testing-on-Kubernetes)
+- [Questions](#questions)
+- [GitHub issues related to this task](#github-issues-related-to-this-task)
+
+## Which Operators do we support
+Operator Hub is not the only place to find/list Operators that could be
+installed on a OpenShift/k8s cluster. There’s also KUDO. However, to start
+things off, we must focus on a limited number of Operators and iterate from
+there. Hence, we will focus only on OperatorHub (kind of obvious 😉).
+
+There are three main categories of Operators as described in the OpenShift
+docs. We could focus on one of those categories or select specific Operators
+from each of the categories.
+
+From a development perspective, we might just take a CRD definition and throw
+it to the Kubernetes API. The controller running in background would take care
+of spinning up the service. So, it wouldn’t matter a lot to differentiate
+between what we support and what we don’t. However, limiting our scope to a few
+Operators will keep us from being overwhelmed.
+
+## Installing Operators
+There’s an open issue about adding capability in odo to install Operators and
+installing “Service Binding Operator” (which helps bind applications with
+Operator-backed services) as a one-off command.
+
+As far as creating services from Service Catalog is concerned, we expect the
+users to enable Service Catalog in their Minishift or OpenShift cluster before
+being able to see any services in `odo catalog list services` output. Why should
+this be different for Operators? We should mention in the documentation about
+using `kube:admin` user for CRC and/or contacting administrator to install
+required Operators in OpenShift cluster.
+
+As for using Service Binding Operator, we also should explore the possibility
+of linking services to components without using the Operator.
+
+## List the services/operators
+Taking this from Tomas’s comment on the issue. Added column for available
```

Comment:
it would be good to include a link to the mentioned comment https://github.com/openshift/odo/issues/2461#issuecomment-566577064